<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MercurialPage</class>
 <widget class="QWidget" name="MercurialPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>751</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Mercurial Interface&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line15">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="title">
      <string>Global Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Encoding:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="encodingComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the encoding to be used by Mercurial</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Encoding Mode:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="encodingModeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the encoding mode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="hiddenChangesetsCheckBox">
        <property name="text">
         <string>Consider hidden changesets</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Log</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>No. of log messages shown:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="logSpinBox">
        <property name="toolTip">
         <string>Enter the number of log messages to be shown</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>999999</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>41</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Commit</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>No. of commit messages to remember:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="commitSpinBox">
        <property name="toolTip">
         <string>Enter the number of commit messages to remember</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Incoming / Outgoing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="logBrowserCheckBox">
        <property name="toolTip">
         <string>Select to show the incoming and outgoing log in a log browser dialog</string>
        </property>
        <property name="text">
         <string>Use Log Browser for incoming / outgoing log</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Pull</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="pullUpdateCheckBox">
        <property name="toolTip">
         <string>Select to update the working directory to new tip</string>
        </property>
        <property name="text">
         <string>Update after pulling</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="preferUnbundleCheckBox">
        <property name="toolTip">
         <string>Select to prefer unbundle over pull</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;Prefer Unbundle&lt;/b&gt;
&lt;p&gt;If this option is selected, it will be checked, if a Mercurial changegroups file from a previous &quot;Show Incoming&quot; request exists. In this case it will be applied and deleted. Otherwise a pull request will be sent.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Prefer Unbundle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Merge</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="internalMergeCheckBox">
        <property name="toolTip">
         <string>Select to enforce usage of the Mercural internal merge tool</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;Internal Merge Tool&lt;/b&gt;
&lt;p&gt;Select to enforce usage of the Mercural internal merge tool,which leaves conflict markers in the file in case of conflicting changes.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Use internal merge with conflict markers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Revert</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="backupCheckBox">
        <property name="toolTip">
         <string>Select to create a backup file (.orig) before reverting changes</string>
        </property>
        <property name="text">
         <string>Create Backup Files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Cleanup</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Pattern:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cleanupPatternEdit">
        <property name="toolTip">
         <string>Enter the file name patterns to be used for cleaning up (entries separated by a space character)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="configButton">
     <property name="toolTip">
      <string>Edit the Mercurial configuration file</string>
     </property>
     <property name="text">
      <string>Edit configuration file</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>388</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>encodingComboBox</tabstop>
  <tabstop>encodingModeComboBox</tabstop>
  <tabstop>hiddenChangesetsCheckBox</tabstop>
  <tabstop>logSpinBox</tabstop>
  <tabstop>commitSpinBox</tabstop>
  <tabstop>logBrowserCheckBox</tabstop>
  <tabstop>pullUpdateCheckBox</tabstop>
  <tabstop>preferUnbundleCheckBox</tabstop>
  <tabstop>internalMergeCheckBox</tabstop>
  <tabstop>backupCheckBox</tabstop>
  <tabstop>cleanupPatternEdit</tabstop>
  <tabstop>configButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
